client/packages/common/src/ui/layout/tables/material-react-table/useNonPaginatedMaterialTable.ts

```diff
@@ -1,14 +1,21 @@
 import { MRT_RowData } from 'material-react-table';
 import { BaseTableConfig, useBaseMaterialTable } from './useBaseMaterialTable';
 
+const VIRTUALIZATION_THRESHOLD = 250;
```

Comment:
I noticed that when the table is virtualized, there's no indication of how many total rows there are, as the scroll bars are only in proportion to the currently virtualized list, not the full table. So it seemed like a good idea to only virtualize when we need to -- is 250 a good threshold you think?

Reply:
Ignore this -- have restored to original behaviour. Turns out that the `enableRowVirtualization` setting only considers the first load, and the `data` is usually undefined on first load, so we don't get a meaningful value. There's things that could be done to handle this, but it's overly complex, and doesn't really matter now that I've fixed that scroll behaviour I showed you.